### PR TITLE
Updated dependecies and fix error in the backend 

### PR DESCRIPTION
**Pull request by zroger49**
Change introduced in broadlink_service.py. Our change allows the discover of devices if one device is locked. 

Also updated python depedencies

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,6 @@
 *.pyc
 output/
 __pycache__
-build
```
Comment (timothystewart6) [inline .gitignore:4]: I think we still need to exclude this from getting committed

File: frontend/src/components/App.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ function App() {
   };
 
   const handleDocumentationClick = () => {
-    window.location = 'https://github.com/techno-tim/techno-broadlink';
+    window.location = 'https://automacaoraceland.pt';
```
Comment (timothystewart6) [inline frontend/src/components/App.js:63]: Can we revert this?

**Review by timothystewart6 — CHANGES_REQUESTED** [review]
Hey!  Thank you so much!  Any way you can address the comments, otherwise 👍

File: frontend/src/components/App.js (hunk 2 above)
Comment (AndreTeixeira1998) [reply frontend/src/components/App.js:63]: oh, of course, I thought I had changed that. I'm so sorry

Comment (timothystewart6) [reply frontend/src/components/App.js:63]: NP!

File: .gitignore [hunk 3]
```diff
@@ -4,4 +4,4 @@ __pycache__
 build
 api/config
 api/src/config
-node_modules
\ No newline at end of file
+node_modules
```
Comment (timothystewart6) [inline .gitignore:7]: Can you revert this too?  We don't want to include `node_modules` :)

File: postman/techno-broadlink.postman_collection.json [hunk 4]
```diff
@@ -152,4 +152,4 @@
 		}
 	],
 	"protocolProfileBehavior": {}
-}
\ No newline at end of file
+}
```
Comment (timothystewart6) [inline postman/techno-broadlink.postman_collection.json:155]: It looks like you might have adjust spacing?  Not sure this commit needs to be here in this PR

**Comment by AndreTeixeira1998** [comment]
I think the changes are made, could you please check?

File: postman/techno-broadlink.postman_collection.json (hunk 4 above)
Comment (timothystewart6) [reply postman/techno-broadlink.postman_collection.json:155]: No worries!

**Comment by timothystewart6** [comment]
Thank you so much!

**Comment by timothystewart6** [comment]
Thank you so much!

**Pull request merged**